java-frontend/src/main/java/org/sonar/java/resolve/JavaType.java

```diff
@@ -62,6 +62,9 @@
   JavaSymbol.TypeJavaSymbol symbol;
 
   public JavaType(int tag, JavaSymbol.TypeJavaSymbol symbol) {
+    Random r = new Random();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a cryptographically strong random number generator (RNG) like "java.security.SecureRandom" in place of this PRNG [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2245)